pkg/translate/translate_value.go

```diff
@@ -267,8 +263,11 @@ func translateHPASpec(inPath string, outPath string, value interface{}, valueTre
 	if !asEnabled {
 		return nil
 	}
-	newP := util.PathFromString(inPath)[0]
-	minPath := newP + ".autoscaleMin"
+
+	newP := util.PathFromString(inPath)
+	// last path element is k8s setting name
+	newPS := newP[:len(newP)-1].String()
+	minPath := newPS + ".autoscaleMin"
```

Comment:
the logic for autoscaleMin/Max should be the same, so i feel like some code below could be de-duped. that may be the case for other params too...